src/components/playback/playbackmanager.js

```diff
@@ -1848,6 +1848,51 @@ class PlaybackManager {
                     SortBy: options.shuffle ? 'Random' : 'SortName',
                     MediaTypes: 'Audio'
                 });
+            } else if (firstItem.Type === 'Series' || firstItem.Type === 'Season') {
+                const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
+                promise = apiClient.getEpisodes(firstItem.SeriesId || firstItem.Id, {
+                    IsVirtualUnaired: false,
+                    IsMissing: false,
+                    UserId: apiClient.getCurrentUserId(),
+                    Fields: 'Chapters'
+                }).then(function (episodesResult) {
+                    let foundItem = false;
+                    const isSeries = firstItem.Type === 'Series'
+                    const originalResults = episodesResult.Items;
+                    episodesResult.Items = episodesResult.Items.filter(function (e) {
+                        if (foundItem) {
+                            return true;
+                        }
+                        if (!e.UserData.Played && (isSeries || e.SeasonId == firstItem.Id)) {
+                            foundItem = true;
+                            return true;
+                        }
+
+                        return false;
+                    });
+
+                    if (episodesResult.Items.length === 0) {
+                        const userId = apiClient.getCurrentUserId()
+                        if (isSeries) {
+                            episodesResult.Items = originalResults;
+                            apiClient.markUnplayed(userId, firstItem.Id, {});
```

Comment:
I don't think we should remove the "Played" mark - the server should do it for us, but right now it doesn't always do it. Moreover, there is a [discussion](https://github.com/jellyfin/jellyfin-meta/discussions/56) about this.

I suggest removing it for now, even though it will cause some inconsistency when rewatching.